components/identity-core/org.wso2.carbon.identity.core/src/test/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilderTest.java

```diff
@@ -260,33 +260,32 @@ public void testBuild() {
 
         return new Object[][]{
                 {"https", "www.wso2.com", 9443, "/proxyContext", "abc", false, null, "", fragmentParams,
-                        "https://www.wso2.com:9443/proxyContext#key1%3Dfragment%26key2%3Dfragment%26key3%3Dfragment" +
-                                "%26key4%3Dfragment", ""},
+                        "https://www.wso2.com:9443/proxyContext#key1=fragment&key2=fragment&key3=fragment&key4" +
```

Comment:
Can we add a test case where a parameter has a special character like "space" or some character that needs encoding?